docs/sources/dashboards/share-dashboards-panels/_index.md

```diff
@@ -298,7 +298,7 @@ Here's an example of what the HTML code might look like:
 ></iframe>
 ```
 
-The result is an interactive Grafana visualization embedded in an iframe.
+The result is an interactive Grafana visualization embedded in an iframe. This is only supported on Grafana OSS. You cannot embed a private or public panel within Grafana Cloud. 
```

Comment:
What exactly do you mean isn't supported here? That you can't embed the dashboard _back into_ your Cloud instance and that you can with OSS (and I'm assuming Enterprise)?

Reply:
Just to clarify that even if a panel in a dashboard is publicly accessible if you try to embed it within an iframe it still won't display without the `allow_embedding` which support largely will not request. The goal is to reduce contact points, make it apparent public vs private doesn't matter.

Reply:
Just to clarify, do you mean it won't display for Cloud users unless they're signed in or _not at all_? If the first, then this bit doesn't help here because it makes it sound like you can't use the feature at all if you're on Cloud.

```suggestion
The result is an interactive Grafana visualization embedded in an iframe.
```

Reply:
Sorry for the confusion! It won't display unless they're signed in which sort of defeats the purpose of embedding a dashboard elsewhere. I like your language better, LGTM! Ty!

Reply:
Understood! That's probably a larger question to bring up with the sharing squad of whether this feature really make sense for Cloud users. It sounds like this is a common issue you're coming up against; do you want me to raise it with the squad?